inc/wp-components/classes/class-social-sharing.php

```diff
@@ -83,7 +83,7 @@ public function get_twitter_component() : Social_Item {
 							'text' => $this->get_title(),
 							'url'  => $this->get_url(),
 						],
-						'https://twitter.com/share/'
+						'https://twitter.com/intent/tweet/'
```

Comment:
When I click one a `https://twitter.com/share/` link as they are currently, it redirects to a  `https://twitter.com/intent/tweet/` URL. But this `intent/tweet` URL is the one that twitter suggests in [their docs](https://developer.twitter.com/en/docs/twitter-for-websites/tweet-button/overview), so it's probably a good idea to change it.